setup.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+
+import os
+import sys
+
+try:
+    from setuptools import setup
+except ImportError:
+    from distutils.core import setup
+
+
+if sys.argv[-1] == 'publish':
+    os.system('python setup.py sdist upload')
+    sys.exit()
+
+readme = open('README.rst').read()
+doclink = """
+Documentation
+-------------
+
+The full documentation is at http://theblues.rtfd.org.
+"""
```

Comment:
No, and while docs need work putting them up on rtfd isn't on the roadmap; I've removed that bit altogether.